### PR TITLE
replased classes router with hooks router, added a parent variable fo…

### PR DESCRIPTION
**Pull request by okarem**
look at mohammads first

replaced classes router with hooks router, added a parent variable formDate in App.js to use down the line in all pages, added some Landing.js component (not finished) to the router

### CODE REVIEW

File: barberia/src/App/pages/Calendar.js [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
 import AppoPicker from "./TimePicker";
-
+import {A} from "hookrouter";
 import "react-datepicker/dist/react-datepicker.css";
 
-function Calendar() {
-  const [startDate, setStartDate] = React.useState(new Date());
+function Calendar({formDate,setFormDate}) {
+  const [startDate, setStartDate] = React.useState(formDate || new Date());
+  console.log("date is ",formDate ,setFormDate);
```
Comment (jamiecoe) [inline barberia/src/App/pages/Calendar.js]: Please can you remove this `console.log`. It's good practice not to leave stray `console.logs` in Production code.

File: barberia/src/App/pages/Home.js [hunk 2]
```diff
@@ -1,19 +1,17 @@
 import React, { Component } from "react";
 import { Link } from "react-router-dom";
-
+import {routes} from "../App";
+import {A} from "hookrouter";
 class Home extends Component {
```
Comment (jamiecoe) [inline barberia/src/App/pages/Home.js]: Any particular reason why this is a Class based Component? 

You've used Functional Components elsewhere, and this is just returning JSX (ie: no state). I'd advice you change this to be a function.

File: barberia/src/App/pages/Landing.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.barberia-name{
+    position: absolute;
```
Comment (jamiecoe) [inline barberia/src/App/pages/Landing.css]: Dodgy formatting! Use Prettier 💅

File: barberia/src/App/pages/TimePicker.js [hunk 4]
```diff
@@ -23,12 +23,13 @@ const AppoPicker = (props) => {
   React.useEffect(() => {
     const onTimeSelect = event => setTime(event.time);
     let picker = new AppointmentPicker(pickerRef.current, options);
```
Comment (jamiecoe) [inline barberia/src/App/pages/TimePicker.js]: Why are you using `let` on `picker` and `currentRef`? These variables are not being mutated, use `const` instead.

File: barberia/src/components/buttons.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
+.hot-shit-button{
+/* position: absolute; */
+width: 232px;
```
Comment (jamiecoe) [inline barberia/src/components/buttons.css]: Please can you remove commented out code? Also Prettier 💅

File: barberia/src/components/buttons.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./buttons.css";
+
+
+const hotShitButton = props => {
```
Comment (jamiecoe) [inline barberia/src/components/buttons.js]: 💩 🔥

File: barberia/src/App/pages/Calendar.js (hunk 1 above)
Comment (okarem) [reply barberia/src/App/pages/Calendar.js]: cleaned

File: barberia/src/App/pages/Home.js (hunk 2 above)
Comment (okarem) [reply barberia/src/App/pages/Home.js]: refactored

File: barberia/src/App/pages/Landing.css (hunk 3 above)
Comment (okarem) [reply barberia/src/App/pages/Landing.css]: reformated

File: barberia/src/App/pages/TimePicker.js (hunk 4 above)
Comment (okarem) [reply barberia/src/App/pages/TimePicker.js]: fixed

File: barberia/src/components/buttons.css (hunk 5 above)
Comment (okarem) [reply barberia/src/components/buttons.css]: done

**Pull request merged**